eduNEXT/edx-platform
# JU/UT003

## Description
Translates some problem examples from YAML files.

## Testing instructions

Log in to Studio and try to create all the basic problems in a course. The instructions for every problem type should be in Spanish.

**Previous version commit:** 069bdbf48f
